pkg/pillar/scripts/device-steps.sh

```diff
@@ -164,6 +167,24 @@ if [ ! -d $PERSISTDIR/log ]; then
     mkdir $PERSISTDIR/log
 fi
 
+
+LAST_EVE_VERSION=""
+if [ -f $LAST_IMAGE_VERSION_FILE ];
+then
+     LAST_EVE_VERSION=$(jq '.Version' < $LAST_IMAGE_VERSION_FILE| tr -d '"')
+fi
+
+# Run upgradeconverter
+if [ "$LAST_EVE_VERSION" = "$EVE_VERSION" ];
+then
+     echo "$(date -Ins -u) device-steps: LAST_EVE_VERSION($LAST_EVE_VERSION) = EVE_VERSION($EVE_VERSION)"
```

Comment:
Odd to log both versions when they are the same. It would be much more interesting to log the two different versions in the else branch, which you do not currently do.

Reply:
Good Catch :-) I had switched the if statements around..Guess I missed changing the prints.. Fixed it..